Frameworks/include/CoreGraphics/DWriteWrapper.h

```diff
@@ -29,12 +29,17 @@
 #import <memory>
 
 // General DWrite helpers
-COREGRAPHICS_EXPORT CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedString);
+#ifdef __cplusplus
+extern "C++" std::wstring _GetUserDefaultLocaleName();
+#endif
```

Comment:
I guess I don't have a better answer for you than 'convenience'. The alternative would be to make the caller allocate a char buffer for us to dump into. Same below, it's more convenient to just pass a shared_ptr around than to define (iirc) a copy constructor for that struct. What are the downsides of cross module exporting c++?

---
In reply to: [90742313](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90742313) [](ancestors = 90742313)